beluga/test/beluga/sensor/test_lfm_with_unknown_space.cpp

```diff
@@ -0,0 +1,271 @@
+// Copyright 2022-2023 Ekumen, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <utility>
+#include <vector>
+
+#include <range/v3/range/conversion.hpp>
+#include <range/v3/view/transform.hpp>
+#include <sophus/common.hpp>
+
+#include "beluga/sensor/likelihood_field_model.hpp"
+#include "beluga/test/general_occupancy_grid.hpp"
+
+namespace {
+
+using beluga::testing::GeneralOccupancyGrid;
+
+using UUT = beluga::LikelihoodFieldModel<GeneralOccupancyGrid<5, 5>>;
+
+TEST(LikelihoodFieldModelUnknownSpace, LikelihoodField) {
+  constexpr double kResolution = 0.5;
+  // clang-format off
+  const auto grid = GeneralOccupancyGrid<5, 5>{{
+    -1 , -1 , -1 , 100, 100,
+    -1 , 0  , 0  , 0  , 100,
+    -1 , 0  , 0  , 0  , 100,
+    100, 0  , 0  , 0  , 100,
+    100, 100, 100, 100, 100},
+    kResolution};
+  // clang-format on
+
+  const double param_mld = 20.0;
+  const double unknown_likelihood = 1 / param_mld;
```

Comment:
@DPR00 nit: consider making these `constexpr` and switching to Hungarion notation e.g. `kUnknownSpaceLikelihood`.

Reply:
Also, please don't use non-obvious abbreviations. I don't know what `param_mld` is. Characters are free. Spell it out in full.